packages/preview-middleware-client/src/flp/init.ts

```diff
@@ -4,8 +4,8 @@ import type { InitRtaScript, RTAPlugin, StartAdaptation } from 'sap/ui/rta/api/s
 import type { RTAOptions } from 'sap/ui/rta/RuntimeAuthoring';
 import IconPool from 'sap/ui/core/IconPool';
 import ResourceBundle from 'sap/base/i18n/ResourceBundle';
-import UriParameters from 'sap/base/util/UriParameters';
-
+import AppState from 'sap/ushell/services/AppState';
+import Localization from 'sap/base/i18n/Localization';
```

Comment:
Please remove this global import. `sap/base/i18n/Localization` is only available since  UI5 version 1.118.